_data/meltano/utilities/dbt-snowflake/dbt-labs.yml

```diff
@@ -0,0 +1,123 @@
+name: dbt-snowflake
+label: dbt Snowflake (EDK preview)
+executable: dbt_invoker
+namespace: dbt_snowflake
+docs: https://docs.meltano.com/guide/transformation
+repo: https://github.com/dbt-labs/dbt-snowflake
+ext_repo: https://github.com/meltano/dbt-ext
+pip_url: dbt-core~=1.3.0 dbt-snowflake~=1.3.0 git+https://github.com/meltano/dbt-ext.git@main
+variant: dbt-labs
+maintenance_status: active
+settings_preamble: |
+  Settings for dbt itself can be configured through [dbt_project.yml](https://docs.getdbt.com/reference/dbt_project.yml) as usual, which can be found at transform/dbt_project.yml in your Meltano project. dbt also has [adapter-specific documentation for Snowflake](https://docs.getdbt.com/reference/resource-configs/snowflake-configs).
+settings:
+- name: skip_pre_invoke
+  label: Skip Pre-invoke
+  kind: boolean
+  value: false
+  description: Whether to skip pre-invoke hooks which automatically run dbt clean and deps
+  env: DBT_EXT_SKIP_PRE_INVOKE
+- name: type
+  label: dbt Profile type
+  env: DBT_EXT_TYPE
+  value: snowflake
+- name: project_dir
+  label: Projects Directory
+  value: $MELTANO_PROJECT_ROOT/transform
+- name: profiles_dir
```

Comment:
@pandemicsyn after running `meltano invoke dbt-snowflake:initialize` I got `$MELTANO_PROJECT_ROOT/transform/transform/profiles.yml` in my repo and this setting is looking for it in `$MELTANO_PROJECT_ROOT/transform/profiles/snowflake` so my `meltano invoke dbt-snowflake debug` failed.

I think this is a quick fix to either update the profiles dir default values or fix the extension file bundle piece. I think the default value makes sense so I'd vote to update he extension itself.

I only tested with snowflake but presumably this would propagate to all the adapters.

Reply:
PR/fix pending for this over in the ext repo - https://github.com/meltano/dbt-ext/pull/8

Reply:
@pnadolny13 fix landed in main on the dbt-ext repo 🚀

Reply:
@pandemicsyn thanks it worked perfectly for me with a fresh install!!